src/status_im/ui2/screens/chat/messages/message.cljs

```diff
@@ -470,7 +468,7 @@
                                   (js/setTimeout #(on-long-press-fn on-long-press message content) 200))
                               (on-long-press-fn on-long-press message content)))]
         (reset! ref on-long-press)
-        [rn/touchable-highlight
+        [rn/touchable-opacity
```

Comment:
could you elaborate on this change, why opacity?

Reply:
<img width="464" alt="Screenshot 2022-10-21 at 10 53 47" src="https://user-images.githubusercontent.com/29354102/197131946-b2f56bf4-b072-4de4-9e6e-8ca445619406.png">
TouchableHighlight causes this black box

Reply:
Probably can be fixed with setting underlay color and opacity. I usually prefer touchableOpacity, is there a reason to use touchableHighlight?

Reply:
probably then its related to rn namespace, is it quo namespace ? probably touchableHighlight is different from one in react namespace

Reply:
i guess `src/status_im/ui2/screens/chat/messages/message.cljs` was the only file i moved without changes from old ui, because i didn't know what changes has been made already in it, so we shouldn't change it, but anyway, it will be reimplemented so for now lets just make it work doesn't matter if it touchableHighlight or touchableOpacity